src/protocols/secure_channel/CASESession.cpp

```diff
@@ -283,6 +249,8 @@ CHIP_ERROR CASESession::EstablishSession(const Transport::PeerAddress peerAddres
     mExchangeCtxt->SetResponseTimeout(kSigma_Response_Timeout);
     mConnectionState.SetPeerAddress(peerAddress);
     mConnectionState.SetPeerNodeId(peerNodeId);
+    mTrustedRootId = operationalCredentialSet->GetTrustedRootId(opCredSetIndex);
```

Comment:
the only use of opCredSetIndex seems to be to `GetTrustedRootId`. Why do we pass the index instead of the trusted root Id itself?

Reply:
This is because the TrustedRootId was removed from the spec. So, we should remove it from the public methods and use an index instead.